mod/network.php

```diff
@@ -456,7 +456,7 @@ function networkFlatView(App $a, $update = 0)
 	$items = q("SELECT %s FROM `item` $sql_post_table %s
 		WHERE %s AND `item`.`uid` = %d
 		ORDER BY `item`.`id` DESC $pager_sql ",
-		item_fieldlists(), item_joins(), item_condition(),
+		item_fieldlists(), item_joins($_SESSION['uid']), item_condition(),
```

Comment:
Why not `local_user()`?

Reply:
That variable is used in the same query for the uid.

Reply:
And it looks wrong as well. Please normalize both references.

Reply:
The `intval` is protecting against edge-case values, and locsl_user() is used both before and after this query, it’s safe to use.